components/RuiBreadcrumbs/src/RuiBreadcrumbs.css.ts

```diff
@@ -0,0 +1,88 @@
+import {css} from 'lit-element';
+
+
+/**
+	*  RuiBreadcrumbs Variables
+ 	*  Added colour to prevent empty block
+*/
+
+export const variables = css`
+	:host {
+		/* BASE STYLES */
+
+		/**
+		 * @variable The base colour for the breadcrumbs
+		 */
+		--color: #000000;
+	
+		/**
+		 * @variable Hover colour for the breadcrumbs
+		 */
+		--hover-color: #01447E;
+	
+		/**
+		 * @variable padding size
+		 */
+		--padding: 8px;
+	}
+    `;
+
+/**
+	* RuiBreadcrumbs CSS
+*/
+
+export const layout = css`
+	/**
+		* css for json array breadcrumbs
+	*/
+
+	nav.crumbs ol {
+		list-style: none;
+	}
+	
+	nav.crumbs li {
+		display: inline;
+	}
+	
+	nav.crumbs li + li::before {
+		padding: var (--padding);
+		color: var(--color);
+		content: "/\\00a0";
+	}
+	
+	nav.crumbs a:hover {
```

Comment:
Can you please also add `:focus` pseudo-class wherever you have `:hover`?
We need to take into account keyboard users